### PR TITLE
37961: disable audit logging by default

### PR DESCRIPTION
**Pull request by cheewaio**

### CODE REVIEW

File: deploy/olm-catalog/ibm-helm-repo-operator/3.6.0/operator.ibm.com_helmrepos_crd.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ metadata:
   name: helmrepos.operator.ibm.com
   labels:
     name: helmrepos.operator.ibm.com
-    app.kubernetes.io/instance: helm-repo
+    app.kubernetes.io/instance: helmrepos.operator.ibm.com
```
Comment (bluzarraga) [inline deploy/olm-catalog/ibm-helm-repo-operator/3.6.0/operator.ibm.com_helmrepos_crd.yaml]: did you mean to change this back to helmrepos.operator.ibm.com?

Comment (cheewaio) [reply deploy/olm-catalog/ibm-helm-repo-operator/3.6.0/operator.ibm.com_helmrepos_crd.yaml]: darn..what's going on

File: helm-charts/helm-repo/templates/deployment.yaml [hunk 2]
```diff
@@ -53,7 +53,7 @@ spec:
       tolerations:
 {{ toYaml .Values.tolerations | indent 8 }}
       containers:
-      {{ if .Values.auditService.config.enabled }}
+      {{ if eq .Values.auditService.config.enabled "true" }}
```
Comment (bitscuit) [inline helm-charts/helm-repo/templates/deployment.yaml]: I think `"true"` shouldn't be surrounded in quotes. Seems to be causing a helm linting error that's affecting PROW

**Review by bitscuit — CHANGES_REQUESTED** [review]

**Review by bitscuit — APPROVED** [review]
/lgtm

**Pull request merged**